ansible/playbooks/tasks/cluster-bootstrap.yaml

```diff
@@ -401,7 +403,7 @@
   when:
     - is_primary
     - reg_vip_location.stdout | trim | split(' ') | last != primary_hostname
-  until: reg_vip_location2.stdout_lines[0] | trim | split(' ') | last == primary_hostname
+  until: reg_vip_location2.stdout_lines | length > 0 and reg_vip_location2.stdout_lines[0] | trim | split(' ') | last == primary_hostname
```

Comment:
This addresses **Problem 3**: it adds a check for output length, so the task retries if the output is not yet ready instead of immediately failing

Reply:
task retry means to run `crm configure primitive rsc_aws_stonith` more than one time? Is it fine from crm point of view?